src/AppSearch/Request/ListSynonymSets.php

```diff
@@ -42,7 +45,7 @@ public function __construct(string $engineName)
 	 */
 	public function setCurrentPage(int $currentPage): self
 	{
-		$this->queryParams['page[current]'] = $currentPage;
+		$this->body['filters']['date']['to'] = $currentPage;
```

Comment:
Does this work correctly if the body is empty? Or is it guaranteed that 'filters' and 'date' will already exist when this runs?

Reply:
If `$this->body` does not exist as array, it will be created. This is [managed by PHP](https://www.php.net/manual/en/language.types.array.php#language.types.array.syntax.modifying) itself.
Moreover, the `body` property type is not defined in the [Request](https://github.com/elastic/enterprise-search-php/blob/7.13/src/Request/Request.php#L46-L49) base class, since it can be potentially also an object.